scvi/dataset/brain_large.py

```diff
@@ -75,12 +71,12 @@ def preprocess(self):
         std_scaler = StandardScaler(with_mean=False)
         std_scaler.fit(matrix.transpose().astype(np.float64))
         subset_genes = np.argsort(std_scaler.var_)[::-1][:self.nb_genes_kept]
-        subsampled_matrix = subsample_genes(gene_bc_matrix, subset_genes, unit_test=self.unit_test)
+        subsampled_matrix = subsample_genes(gene_bc_matrix, subset_genes, unit_test=(self.save_path == 'tests/data/'))
         print("%d genes subsampled" % subsampled_matrix.matrix.shape[0])
         print("%d cells subsampled" % subsampled_matrix.matrix.shape[1])
 
         # Extracting batch indices
-        if not self.unit_test:
+        if not self.save_path == 'tests/data/':
```

Comment:
Can we avoid treating the test data differently than the real data here? Perhaps by adding barcoes to the test data?